feedback-suggestion/src/app/endpoints/authenticated_request.py

```diff
@@ -20,7 +20,7 @@ def get_artemis_version(server):
         response = requests.get(server)
         if not response.ok:
             raise ConnectionError("Connection to artemis server failed")
-        if "Content-Version" in response.headers:
+        if hasattr(response.headers, "Content-Version"):
```

Comment:
Isn't headers a dictionary, hence should be tested with in?

Reply:
`hasattr` probably won't work. We had made this error in a previous version of our code, maybe this is a copypaste-error.